kuma/scrape/scraper.py

```diff
@@ -36,6 +36,12 @@ def session(self):
         return self._session
 
     def request(self, path, raise_for_status=True, method='GET'):
+        # When doing a long-running recursive scrape, you may find
+        # it useful to uncomment the following two lines. The sleep
+        # helps to avoid rate-limiting, and the print allows you to
+        # monitor the progress of your scrape.
+        # time.sleep(1)
```

Comment:
Is it faster to `sleep(1)` for each request, rather than hit the rate limit and wait for it to clear? If so, we could try to add some smarter self-limiting.

Reply:
It seemed faster to me, but maybe just becasue of the print statement so I could see the steady progress. The 1 minute sleeps were a drag.

Not a change I was confident enough in to uncomment, and not something that seems worth spending more time on. So I could also just remove the comment entirely and leave the file unmodified.